### ISSUE TITLE
Broken link in documentation

### ISSUE THREAD

**Issue by maxrudolph** [jupyter/nbgrader#726]
Looking at the nbgrader documentation here:
http://nbgrader.readthedocs.io/en/stable/configuration/jupyterhub_config.html#running-jupyterhub-and-formgrader

The example and link referenced no longer appear to exist:
```
For a minimal working example of using JupyterHub with SSL, please see the nbgrader.tests.formgrader.manager.HubAuthSSLManager in the nbgrader tests.
```
https://github.com/jupyter/nbgrader/blob/master/nbgrader/tests/formgrader/manager.py

**Comment by lgpage** [comment]
Hmm, the "0.4.x" version on RTD has the correct link from PR #709 and the "latest" version shows the updates of formgrader being an extension. So the "stable" version should will be fixed with the next patch release v0.4.1 of nbgrader.

**Comment by jhamrick** [comment]
Oh, ok, great! Then we can close this.
